tools/install/Makefile.skel

```diff
@@ -16,16 +16,23 @@
 #
```

Comment:
Please move the #!/bin/sh to the very first line above the copyright, as that's the only place the kernel will ever look.

Reply:
Ok, that is also for all other bash scripts then.

Reply:
Ah yes, true!